db/migrate/20171213161035_add_indexes_on_images.rb

```diff
@@ -0,0 +1,25 @@
+class AddIndexesOnImages < ActiveRecord::Migration[5.0]
+  def up
+    deduped = Image.group(:name, :compute_resource_id, :operatingsystem_id).pluck('min(id)')
+    Image.where.not(id: deduped).each_with_index do |image, i|
+      say "Found images with duplicate name #{image.name} on Compute Resource #{ComputeResource.unscoped.find(image.compute_resource_id)}!"
+      say "Duplicates have been renamed, please check your setup to resolve the duplication."
```

Comment:
@tbrisker I think it would be more friendly to tell the user where to check for this - e.g: Please check "https://foremanfqdn/compute_resources/id-nameofcomputeresource/images" to rename the duplicate images - as far as I know we have all of the information to show the link here don't we?

Reply:
@dLobatog Thanks, I tried to add this now but since this runs in a migration it seems that we don't have the hostname for the server at this point, `Rails.application.routes.url_helpers.compute_resource_url` fails with: 
`ArgumentError: Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true`

Reply:
I guess we could at least print compute resource name, migration/installer will be run by Foreman maintainer, they should know how to check the specific compute resource.

Reply:
@ares we already do that, compute resource's `to_s` returns the resource name.

Reply:
ah, I was looking at this line only :-), works for me, any more comments @dLobatog ?